app/login/page.tsx

```diff
@@ -1,10 +1,178 @@
-import Users from '../../components/module-components/users/Users';
+'use client';
+
+import Link from 'next/link';
+import Image from 'next/image';
+import { FormikHelpers, useFormik } from 'formik';
+import { toast } from 'react-toastify';
+import * as Yup from 'yup';
+import { useState } from 'react';
+import Turnstile from 'react-turnstile';
+import styles from './login.module.scss';
+import linkedin from '../../public/icons/linkedin.svg';
+import github from '../../public/icons/github.svg';
+import google from '../../public/icons/google.svg';
+import { useTheme } from '../context/themeContext';
+import TextInput from '../../components/reusable-components/text-input/TextInput';
+import { submitLoginForm } from './SubmitLoginForm';
+
+interface FormValues {
+  email: string;
+  password: string;
+}
 
 const Login = () => {
+  const { theme } = useTheme();
+  const [turnstileToken, setTurnstileToken] = useState<string | null>(null);
+  const isLightTheme = theme === 'light';
+  const initialValues = { email: '', password: '' };
+  const validationSchema = Yup.object({
+    email: Yup.string()
+      .email('*Невалидна емаил адреса')
+      .required('*Задолжително внесете емаил адреса'),
+    password: Yup.string()
+      .required('Задолжително внесете пасворд.')
+      .min(8, 'Пасвордот би требало да содржи минимум 8 знаци.')
+      .matches(
+        /^(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]*$/,
+        'Мора да содржи еден број и еден посебен знак'
+      ),
+  });
+  const handleSubmit = async (values: FormValues, { resetForm }: FormikHelpers<FormValues>) => {
+    if (!turnstileToken) {
+      return;
+    }
+
+    const formValues = {
+      email: values.email,
+      password: values.password,
+      cfTurnstileResponse: turnstileToken,
+    };
+
+    try {
+      const response = await submitLoginForm(formValues);
+      if (response) {
+        toast.success(response);
+
+        resetForm();
+      } else {
+        toast.error(response);
+      }
+    } catch (error) {
+      toast.error('Грешка');
+    }
+  };
+
+  const formik = useFormik({
+    initialValues,
+    validationSchema,
+    onSubmit: handleSubmit,
+  });
+
   return (
-    <div>
-      <div>LogIn</div>
-      <Users />
+    <div
+      className={`${styles.loginPage} ${isLightTheme ? styles.lightLoginPage : styles.darkLoginPage}`}
+    >
+      <div className={styles.container}>
+        <div className={styles.welcomeContainer}>
+          <h1
+            className={`${styles.welcomeBackTitle}  ${isLightTheme ? styles.welcomeBackTitleLight : styles.welcomeBackTitleDark}`}
+          >
+            Добредојде назад, Марија!
```

Comment:
we can't know the name here, I wrote to the design team and we will update this for sure.